### PR TITLE
feat: migrated < Paragrapgh />, < MultipleStoryElementsWrapper />, < Link /> to scss modules

### LINKED ISSUE #44: CSS modules migration: < Paragrapgh />, < MultipleStoryElementsWrapper />, < Link />
[Monday ticket](https://monday.monday.com/boards/3532714909/pulses/5294619999)

[CSS modules migration guideline](https://github.com/mondaycom/vibe-storybook-components/blob/master/CSS_MODULES_MIGRATION.md)

### PR DESCRIPTION
**Pull request by Dhoni77**
Fixes #44

### CODE REVIEW

File: src/components/link/link.jsx [hunk 1]
```diff
@@ -1,21 +1,17 @@
 import React from 'react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import { BEMClass } from '../../helpers/utils/bem-helper';
 import CoreLink from '../../helpers/components/Link/Link';
-import './link.scss';
-
-const BASE_CLASS = 'vibe-sb-comps-link';
-const bemHelper = BEMClass(BASE_CLASS);
+import styles from './link.scss';
```
Comment (SergeyRoyt) [inline src/components/link/link.jsx:5]: Only forgot to rename imported file - should be `link.module.scss`

File: src/components/multiple-story-elements-wrapper/multiple-story-elements-wrapper.jsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import cx from 'classnames';
-import './multiple-story-elements-wrapper.scss';
+import styles from './multiple-story-elements-wrapper.scss';
```
Comment (SergeyRoyt) [inline src/components/multiple-story-elements-wrapper/multiple-story-elements-wrapper.jsx:3]: Same here

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Looks great!
Let's only fix a few notes and we are done here

File: src/components/link/link.jsx (hunk 1 above)
Comment (Dhoni77) [reply src/components/link/link.jsx]: done👍

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
All great, thanks!
If it's not too complicated let's only fix the merge conflict! 🙏

**Comment by Dhoni77** [comment]
> All great, thanks! If it's not too complicated let's only fix the merge conflict! 🙏

done @SergeyRoyt 👍

**Review by SergeyRoyt — APPROVED** [review]
Great job, thanks!!

**Pull request merged**